src/app/(BottomTabNavigation)/Profile/DeleteAccount/index.tsx

```diff
@@ -0,0 +1,46 @@
+//import { router } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import {
+  deleteCurrentUser,
+  getCurrentUserInfo,
+  signOutUser,
+} from '../../../../supabase/queries/auth';
+import styles from '../styles';
+
+function DeleteAccountScreen() {
+  const [currSession, setCurrSession] = useState<any>(null);
+  useEffect(() => {
+    getCurrentUserInfo().then(result => {
+      if (result) {
+        const userInfo = {
+          email: result.email,
+          fullName: result.user_metadata.fullName,
```

Comment:
I believe we no longer store `fullName` and only first, middle, and last. When I created a new account the `raw_user_meta_data` looked like this:
`{"address":"Delete Me ","lastName":"Account","firstName":"New","middleName":"New "}`